.gitignore

```diff
@@ -4,3 +4,4 @@
 .kitchen/
 .kitchen.local.yml
 Gemfile.lock
+vendor/bundle/
```

Comment:
Yeah. Just using vendor/ was too general since it was matching files/default/*vendor*/. This now just matches the `./vendor/bundle` path that you get when using Bundler to install local gem copies for test purposes.